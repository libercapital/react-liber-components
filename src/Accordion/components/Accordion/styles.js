import styled from 'styled-components';
import { media } from '../../../Util';
import { handleThemeFromObject } from '../../../Themes';

export const AccordionBox = styled.div`
  overflow: hidden;
  width: 100%;
  background-color: ${props =>
    props.open
      ? handleThemeFromObject(props.theme, 'accordion.boxBackgroundOpen', '#fff')
      : handleThemeFromObject(props.theme, 'accordion.boxBackgroundClosed', '#f9f9f9')};
  border: ${props =>
    props.open
      ? `1px solid ${handleThemeFromObject(props.theme, 'accordion.boxBorderOpen', '#fff')}`
      : `solid 1px ${handleThemeFromObject(
          props.theme,
          'accordion.boxBorderClosed',
          'rgba(150, 150, 150, 0.18)',
        )}`};
  transition: all 400ms ${props => (props.open ? 'ease-in' : 'ease-out')};
  &:hover {
    background-color: ${props =>
      props.open
        ? handleThemeFromObject(props.theme, 'accordion.boxBackgroundOpen', '#fff')
        : handleThemeFromObject(props.theme, 'accordion.boxBackgroundHover', '#f3f3f3')};
  }
`;

export const ShowBox = styled.div`
  max-height: ${props => (props.open ? props.size : '0px')};
  transition: max-height 400ms ${props => (props.open ? 'ease-in' : 'ease-out')};
  overflow: hidden;
`;

export const AccordionAction = styled.div`
  padding: 20px 50px;
  cursor: pointer;
  box-sizing: border-box;
  display: inline-block;
  background-color: ${props => (props.open ? props.color || 'transparent' : 'transparent')};
  width: 100%;
  transition: all 400ms ${props => (props.open ? 'ease-in' : 'ease-out')};
  position: relative;

  ${media.large`
    padding: 30px 50px;
  `}

  &:after {
    position: absolute;
    top: calc(50% - 4px);
    left: 15px;
    width: 0;
    height: 0;
    padding: 0;
    content: '';
    border-left: 7px solid transparent;
    border-right: 7px solid transparent;
    border-top: 8px solid
      ${props =>
        props.open
          ? handleThemeFromObject(props.theme, 'accordion.actionBorderTopOpen', '#009dff')
          : handleThemeFromObject(props.theme, 'accordion.actionBorderTopClosed', '#4e4e4e')};
    pointer-events: none;
    transform: ${props => (props.open ? 'rotate(0deg)' : 'rotate(-90deg)')};
    transition: all 400ms ${props => (props.open ? 'ease-in' : 'ease-out')};
  }
`;
