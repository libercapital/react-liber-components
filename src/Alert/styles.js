import styled, { css } from 'styled-components';
import { MdClose } from '../Icons/MaterialDesign';

const icon = css`
  --size: 24px;
  min-width: var(--size);
  max-width: var(--size);
  min-height: var(--size);
  max-height: var(--size);
`;

export const CloseIcon = styled(MdClose)`
  ${icon}
  &:hover {
    cursor: pointer;
  }
`;

export const AlertBox = styled.div`
  display: flex;
  flex-direction: row;
  align-items: center;
  width: 100%;
`;

export const AlertContent = styled.div`
  display: flex;
  flex-grow: 1;
  flex-direction: column;
  padding: 24px 0;
`;

export const AlertInfoContent = styled.div`
  font-family: Roboto;
  font-size: 16px;
  font-weight: 500;
  line-height: 1.38;
  letter-spacing: normal;
  color: #405f71;
`;
export const AlertAdditionalInfoContent = styled.div`
  font-family: Roboto;
  font-size: 16px;
  font-weight: normal;
  line-height: 1.38;
  letter-spacing: 0px;
  color: #587383;
  padding-top: 4px;
`;

export const AlertIcon = styled.div`
  display: flex;
  flex-grow: 0;
  align-items: center;
  padding: 24px;
`;
