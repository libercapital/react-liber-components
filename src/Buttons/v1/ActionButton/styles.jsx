import styled from 'styled-components';
import { media } from '../../../Util';


export const Label = styled.div`
  text-align: center;
  width: unset;
  opacity: 1;
  transition: all 300ms  ${props => (props.hover ? 'ease-out' : 'ease-in')};
  white-space: nowrap;
  color: inherit;

  ${media.large`
    width: ${props => (props.hover ? props.size || '0px' : '0px')};
    opacity: ${props => (props.hover ? '1' : '0')};
  `}
  ${media.extraLarge`
    width: ${props => (props.hover ? props.size || '0px' : '0px')};
    opacity: ${props => (props.hover ? '1' : '0')};
  `}
`;

export const Icon = styled.div`
  margin-right: 15px;
  transition: margin 300ms  ${props => (props.hover ? 'ease-out' : 'ease-in')};
  ${media.large`
    margin-right: ${props => (props.hover ? '15px' : '0px')};
  `}
  ${media.extraLarge`
    margin-right: ${props => (props.hover ? '15px' : '0px')};
  `}
`;
