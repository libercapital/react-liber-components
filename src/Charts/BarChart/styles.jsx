import styled from 'styled-components';
import { handleThemeFromObject } from 'Themes';

export const empty = null;

export const Slider = styled.input`
  width: ${props => (props.width ? `${props.width}px` : '100%')};
  appearance: none;
  height: 8.5px;
  background-color: ${props => handleThemeFromObject(props.theme, 'barChart.slider.background', '#fff')};
  outline: none;
  border-radius: 4.25px;
  border: solid 2px ${props => handleThemeFromObject(props.theme, 'barChart.slider.border', '#dfe3eb')};

  &::-webkit-slider-thumb {
    -webkit-appearance: none;
    border: 0px;
    width: 55px;
    height: 6px;
    border-radius: 3px;
    background-color: ${props => handleThemeFromObject(props.theme, 'barChart.slider.border', '#dfe3eb')};;
  }

  &::-moz-range-thumb {
    border: 0px;
    width: 55px;
    height: 6px;
    border-radius: 3px;
    background-color: ${props => handleThemeFromObject(props.theme, 'barChart.slider.border', '#dfe3eb')};;
  }

  &::-ms-thumb {
    border: 0px;
    width: 55px;
    height: 6px;
    border-radius: 3px;
    background-color: ${props => handleThemeFromObject(props.theme, 'barChart.slider.border', '#dfe3eb')};;
  }

  &:focus {
    outline: none;
  }
  &::-moz-focus-outer {
    border: 0;
  }
  &::-moz-range-track {
    background-color: ${props => handleThemeFromObject(props.theme, 'barChart.slider.background', '#fff')};
  }
`;

Slider.defaultProps = {
  type: 'range',
  valueAsNumber: true,
};
