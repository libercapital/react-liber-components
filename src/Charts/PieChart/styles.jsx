import styled from 'styled-components';
import { Slice as SliceRaw, VictoryContainer } from 'victory';

export const Slice = styled(SliceRaw)`
  transition: all 300ms;
  cursor: pointer;
`;

export const PieBox = styled(VictoryContainer)`
  width: unset !important;
  height: unset !important;
  & > svg {
    width: ${props => (props.width ? `${props.width}px` : 'unset')} !important;
    height: ${props => (props.height ? `${props.height}px` : 'unset')} !important;
  }
`;
