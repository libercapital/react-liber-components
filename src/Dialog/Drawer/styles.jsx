import styled from 'styled-components';
import { handleThemeFromObject } from '../../Themes';

const getContentPositioning = ({ position, size, fitScreen, drawerSize }) => {
  let width;
  let height;
  switch (position) {
    case 'right':
      width = fitScreen ? '100vw' : `${size}px`;
      height = drawerSize ? `${drawerSize}px` : '100vh';
      return `
        height: ${height};
        width: ${width};
        top: ${drawerSize ? null : '0'};
        right: 0;
        border-radius: ${drawerSize ? '4px 0px 0px 4px' : null};
      `;
    case 'top':
      height = fitScreen ? '100vh' : `${size}px`;
      width = drawerSize ? `${drawerSize}px` : '100vw';
      return `
        height: ${height};
        width: ${width};
        top: 0;
        left: ${drawerSize ? null : '0'};
        border-radius: ${drawerSize ? '0px 0px 4px 4px' : null};
    `;
    case 'bottom':
      height = fitScreen ? '100vh' : `${size}px`;
      width = drawerSize ? `${drawerSize}px` : '100vw';
      return `
        height: ${height};
        width: ${width};
        bottom: 0;
        left: ${drawerSize ? null : '0'};
        border-radius: ${drawerSize ? '4px 4px 0px 0px' : null};
    `;
    case 'left':
    default:
      width = fitScreen ? '100vw' : `${size}px`;
      height = drawerSize ? `${drawerSize}px` : '100vh';
      return `
        height: ${height};
        width: ${width};
        top: ${drawerSize ? null : '0'};
        left: 0;
        border-radius: ${drawerSize ? '0px 4px 4px 0px' : null};
      `;
  }
};

export const Content = styled.div`
  position: fixed;
  height: 100vh;
  background-color: ${props => handleThemeFromObject(props.theme, 'dialog.card', '#fff')};
  ${props => getContentPositioning(props)}
  pointer-events: all;
  box-shadow: 0px 8px 10px -5px rgba(0, 0, 0, 0.2), 0px 16px 24px 2px rgba(0, 0, 0, 0.14),
    0px 6px 30px 5px rgba(0, 0, 0, 0.12);
`;
