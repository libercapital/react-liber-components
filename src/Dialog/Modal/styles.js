import styled, { css } from 'styled-components';
import { Card as RawCard } from '../../GenericStyles';
import { handleThemeFromObject } from '../../Themes';
import { FaTimes } from '../../Icons';

export const Card = styled(RawCard)`
  position: relative;
  ${({ fullscreen }) =>
    fullscreen
      ? css`
          height: 100vh !important;
          width: 100vw !important;
        `
      : css`
          max-width: 80vw;
          max-height: 80vh;
          min-width: 10vw;
          min-height: 10vh;
          height: unset;
        `}
  overflow: ${props => (props.overflow ? props.overflow : 'unset')};
  border-radius: 4px;
  box-shadow: 0 4px 15px 0 rgba(0, 0, 0, 0.2);
  background-color: ${props => handleThemeFromObject(props.theme, 'dialog.card', '#ffffff')};
  margin: 0px !important;
`;

export const CardHeader = styled.div`
  padding: 16px 24px;
  white-space: nowrap;
  font-family: inherit;
  font-size: 20px;
  font-weight: bold;
  text-align: ${props => props.labelAlign};
  color: ${props => handleThemeFromObject(props.theme, 'dialog.header', '#717171')};
  border-bottom: 1px solid
    ${props => handleThemeFromObject(props.theme, 'dialog.separator', '#e6e6e6')};
`;

export const CloseIcon = styled(FaTimes)`
  width: 23px;
  height: 23px;
  fill: ${props => handleThemeFromObject(props.theme, 'dialog.header', '#717171')};
`;

export const CloseButton = styled.div`
  position: absolute;
  padding: ${props => props.closeButtonPadding};
  top: 0;
  right: 0;
  cursor: pointer;
`;
