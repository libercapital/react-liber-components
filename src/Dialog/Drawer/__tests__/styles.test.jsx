import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';
import 'jest-styled-components';
import { Content } from '../styles';

describe('Testing Checkbox Styles', () => {
  configure({ adapter: new Adapter() });

  describe('Content', () => {
    it('should render correctly without props', () => {
      const wrapper = shallow(<Content />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });
    it('should render correctly with theme', () => {
      const theme = {
        dialog: {
          background: '#aaa',
        },
      };
      const wrapper = shallow(<Content theme={theme} />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });
    it('should render correctly with position=left, size=100 and fitScreen=false', () => {
      const props = {
        position: 'left',
        size: 100,
        fitScreen: false,
      };
      const wrapper = shallow(<Content {...props} />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });
    it('should render correctly with position=left, size=100 and fitScreen=true', () => {
      const props = {
        position: 'left',
        size: 100,
        fitScreen: true,
      };
      const wrapper = shallow(<Content {...props} />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });
    it('should render correctly with position=right, size=100 and fitScreen=false', () => {
      const props = {
        position: 'right',
        size: 100,
        fitScreen: false,
      };
      const wrapper = shallow(<Content {...props} />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });
    it('should render correctly with position=right, size=100 and fitScreen=true', () => {
      const props = {
        position: 'right',
        size: 100,
        fitScreen: true,
      };
      const wrapper = shallow(<Content {...props} />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });
    it('should render correctly with position=top, size=100 and fitScreen=false', () => {
      const props = {
        position: 'top',
        size: 100,
        fitScreen: false,
      };
      const wrapper = shallow(<Content {...props} />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });
    it('should render correctly with position=top, size=100 and fitScreen=true', () => {
      const props = {
        position: 'top',
        size: 100,
        fitScreen: true,
      };
      const wrapper = shallow(<Content {...props} />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });
    it('should render correctly with position=bottom, size=100 and fitScreen=false', () => {
      const props = {
        position: 'bottom',
        size: 100,
        fitScreen: false,
      };
      const wrapper = shallow(<Content {...props} />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });
    it('should render correctly with position=bottom, size=100 and fitScreen=true', () => {
      const props = {
        position: 'bottom',
        size: 100,
        fitScreen: true,
      };
      const wrapper = shallow(<Content {...props} />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });
});
