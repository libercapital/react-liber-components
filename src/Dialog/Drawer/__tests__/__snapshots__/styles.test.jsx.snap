// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Checkbox Styles Content should render correctly with position=bottom, size=100 and fitScreen=false 1`] = `
<StyledComponent
  fitScreen={false}
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "styles__Content-sc-54hecm-0",
        "isStatic": false,
        "rules": Array [
          "position:fixed;height:100vh;background-color:",
          [Function],
          ";",
          [Function],
          " pointer-events:all;box-shadow:0px 8px 10px -5px rgba(0,0,0,0.2),0px 16px 24px 2px rgba(0,0,0,0.14),0px 6px 30px 5px rgba(0,0,0,0.12);",
        ],
      },
      "displayName": "styles__Content",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "styles__Content-sc-54hecm-0",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
  position="bottom"
  size={100}
/>
`;

exports[`Testing Checkbox Styles Content should render correctly with position=bottom, size=100 and fitScreen=true 1`] = `
<StyledComponent
  fitScreen={true}
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "styles__Content-sc-54hecm-0",
        "isStatic": false,
        "rules": Array [
          "position:fixed;height:100vh;background-color:",
          [Function],
          ";",
          [Function],
          " pointer-events:all;box-shadow:0px 8px 10px -5px rgba(0,0,0,0.2),0px 16px 24px 2px rgba(0,0,0,0.14),0px 6px 30px 5px rgba(0,0,0,0.12);",
        ],
      },
      "displayName": "styles__Content",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "styles__Content-sc-54hecm-0",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
  position="bottom"
  size={100}
/>
`;

exports[`Testing Checkbox Styles Content should render correctly with position=left, size=100 and fitScreen=false 1`] = `
<StyledComponent
  fitScreen={false}
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "styles__Content-sc-54hecm-0",
        "isStatic": false,
        "rules": Array [
          "position:fixed;height:100vh;background-color:",
          [Function],
          ";",
          [Function],
          " pointer-events:all;box-shadow:0px 8px 10px -5px rgba(0,0,0,0.2),0px 16px 24px 2px rgba(0,0,0,0.14),0px 6px 30px 5px rgba(0,0,0,0.12);",
        ],
      },
      "displayName": "styles__Content",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "styles__Content-sc-54hecm-0",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
  position="left"
  size={100}
/>
`;

exports[`Testing Checkbox Styles Content should render correctly with position=left, size=100 and fitScreen=true 1`] = `
<StyledComponent
  fitScreen={true}
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "styles__Content-sc-54hecm-0",
        "isStatic": false,
        "rules": Array [
          "position:fixed;height:100vh;background-color:",
          [Function],
          ";",
          [Function],
          " pointer-events:all;box-shadow:0px 8px 10px -5px rgba(0,0,0,0.2),0px 16px 24px 2px rgba(0,0,0,0.14),0px 6px 30px 5px rgba(0,0,0,0.12);",
        ],
      },
      "displayName": "styles__Content",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "styles__Content-sc-54hecm-0",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
  position="left"
  size={100}
/>
`;

exports[`Testing Checkbox Styles Content should render correctly with position=right, size=100 and fitScreen=false 1`] = `
<StyledComponent
  fitScreen={false}
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "styles__Content-sc-54hecm-0",
        "isStatic": false,
        "rules": Array [
          "position:fixed;height:100vh;background-color:",
          [Function],
          ";",
          [Function],
          " pointer-events:all;box-shadow:0px 8px 10px -5px rgba(0,0,0,0.2),0px 16px 24px 2px rgba(0,0,0,0.14),0px 6px 30px 5px rgba(0,0,0,0.12);",
        ],
      },
      "displayName": "styles__Content",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "styles__Content-sc-54hecm-0",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
  position="right"
  size={100}
/>
`;

exports[`Testing Checkbox Styles Content should render correctly with position=right, size=100 and fitScreen=true 1`] = `
<StyledComponent
  fitScreen={true}
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "styles__Content-sc-54hecm-0",
        "isStatic": false,
        "rules": Array [
          "position:fixed;height:100vh;background-color:",
          [Function],
          ";",
          [Function],
          " pointer-events:all;box-shadow:0px 8px 10px -5px rgba(0,0,0,0.2),0px 16px 24px 2px rgba(0,0,0,0.14),0px 6px 30px 5px rgba(0,0,0,0.12);",
        ],
      },
      "displayName": "styles__Content",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "styles__Content-sc-54hecm-0",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
  position="right"
  size={100}
/>
`;

exports[`Testing Checkbox Styles Content should render correctly with position=top, size=100 and fitScreen=false 1`] = `
<StyledComponent
  fitScreen={false}
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "styles__Content-sc-54hecm-0",
        "isStatic": false,
        "rules": Array [
          "position:fixed;height:100vh;background-color:",
          [Function],
          ";",
          [Function],
          " pointer-events:all;box-shadow:0px 8px 10px -5px rgba(0,0,0,0.2),0px 16px 24px 2px rgba(0,0,0,0.14),0px 6px 30px 5px rgba(0,0,0,0.12);",
        ],
      },
      "displayName": "styles__Content",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "styles__Content-sc-54hecm-0",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
  position="top"
  size={100}
/>
`;

exports[`Testing Checkbox Styles Content should render correctly with position=top, size=100 and fitScreen=true 1`] = `
<StyledComponent
  fitScreen={true}
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "styles__Content-sc-54hecm-0",
        "isStatic": false,
        "rules": Array [
          "position:fixed;height:100vh;background-color:",
          [Function],
          ";",
          [Function],
          " pointer-events:all;box-shadow:0px 8px 10px -5px rgba(0,0,0,0.2),0px 16px 24px 2px rgba(0,0,0,0.14),0px 6px 30px 5px rgba(0,0,0,0.12);",
        ],
      },
      "displayName": "styles__Content",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "styles__Content-sc-54hecm-0",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
  position="top"
  size={100}
/>
`;

exports[`Testing Checkbox Styles Content should render correctly with theme 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "styles__Content-sc-54hecm-0",
        "isStatic": false,
        "rules": Array [
          "position:fixed;height:100vh;background-color:",
          [Function],
          ";",
          [Function],
          " pointer-events:all;box-shadow:0px 8px 10px -5px rgba(0,0,0,0.2),0px 16px 24px 2px rgba(0,0,0,0.14),0px 6px 30px 5px rgba(0,0,0,0.12);",
        ],
      },
      "displayName": "styles__Content",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "styles__Content-sc-54hecm-0",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
  theme={
    Object {
      "dialog": Object {
        "background": "#aaa",
      },
    }
  }
/>
`;

exports[`Testing Checkbox Styles Content should render correctly without props 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "styles__Content-sc-54hecm-0",
        "isStatic": false,
        "rules": Array [
          "position:fixed;height:100vh;background-color:",
          [Function],
          ";",
          [Function],
          " pointer-events:all;box-shadow:0px 8px 10px -5px rgba(0,0,0,0.2),0px 16px 24px 2px rgba(0,0,0,0.14),0px 6px 30px 5px rgba(0,0,0,0.12);",
        ],
      },
      "displayName": "styles__Content",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "styles__Content-sc-54hecm-0",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
